platforms/Windows/bundle/theme.xml

```diff
@@ -73,7 +73,10 @@
             <Checkbox Name="OptionsInstallAndroidSDKAMD64" X="210" Y="363" Width="-11" Height="17" TabStop="yes" FontId="3" EnableCondition="OptionsInstallAndroidPlatform">#(loc.Sdk_ProductName_Android_amd64)</Checkbox>
             <Checkbox Name="OptionsInstallAndroidSDKARM" X="210" Y="381" Width="-11" Height="17" TabStop="yes" FontId="3" EnableCondition="OptionsInstallAndroidPlatform">#(loc.Sdk_ProductName_Android_armv7)</Checkbox>
             <Checkbox Name="OptionsInstallAndroidSDKX86" X="210" Y="399" Width="-11" Height="17" TabStop="yes" FontId="3" EnableCondition="OptionsInstallAndroidPlatform">#(loc.Sdk_ProductName_Android_x86)</Checkbox>
-
+            <Checkbox Name="OptionsInstallToolchain" X="192" Y="417" Width="-11" Height="17" TabStop="yes" FontId="3" EnableCondition="0">#(loc.Toolchain_ProductName)</Checkbox>
+            <Checkbox Name="OptionsInstallAssertsToolchain" X="210" Y="435" Width="-11" Height="17" TabStop="yes" FontId="3" EnableCondition="OptionsIncludeNoAsserts and OptionsInstallNoAssertsToolchain">#(loc.Asserts_Toolchain_ProductName)</Checkbox>
```

Comment:
This condition still doesn't make sense to me - why does installing the asserts enabled toolchain require the no asserts toolchain?

Reply:
If the NoAsserts package is included in the msi (`OptionsIncludeNoAsserts`), and you check the NoAsserts variant checkbox (`OptionsInstallNoAssertsToolchain`) you are allowed to **uncheck** the Asserts variant (`OptionsInstallAssertsToolchain`).

The oposite is true as well. if the NoAsserts package are not included in the msi, or you did not check the NoAsserts, you can not uncheck the Asserts version. 

There has to be **one** toolchain variant selected.

Reply:
here is a video demonstrating how this should work:

https://github.com/user-attachments/assets/6095b45c-1e99-41b0-970b-3059167ee99e

Reply:
hm, should we default to the +Asserts and only allow selection/de-selection of the NoAsserts if included?